src/types/pivot.ts

```diff
@@ -25,13 +25,14 @@ export type Granularity =
   | "quarter_number";
 
 export interface PivotCoreDimension {
-  name: string;
+  fieldName: string;
   order?: "asc" | "desc";
   granularity?: Granularity | string;
 }
 
 export interface PivotCoreMeasure {
-  name: string;
+  id: string; // Identifier of the measure, `technicalName:aggregator{:autoIncrementedNumber}`.
```

Comment:
can be worth adding to the comment that it's used to identify measures in formulas